### PR TITLE
Fix placement and style of the "Make Floating" button

### PR DESCRIPTION
**Pull request by ajreckof**
Fixes smix8, Calinou, myself and many other

This is a PR about the new make floating button and its position. He is particularly frustrating as we tend to click on it while thinking we will be clicking on the Search Help button that was previously placed exactly at the same position. 

To reduce this problem the button was reduced to its icon and multiple position were tested.

Current PR implements it like that : 
image

Another suggestion was : 
image

Thanks to @MewPurPur for the new svg icon

### CODE REVIEW

**Comment by smix8** [comment]
> Fixes smix8 ...

Am I broken? :(
It takes me a while to really despise something but that Button made it quickly to the top of my chart.

**Comment by YeldhamDev** [comment]
> Am I broken? :(

Well, it also fixes Calinou, themselves, and others, so whatever it is you're not alone. :smile:

**Review by trollodel — COMMENTED** [review]
About this problem, I did some mockups [here](https://github.com/godotengine/godot/pull/62378#issuecomment-1320295599)

Personally, I prefer the OP alternative suggestion, but the current implementation is good too.

The only thing that worries me is that the Make Floating button of docks is now only an icon.

File: editor/plugins/script_editor_plugin.cpp [hunk 1]
```diff
@@ -4025,6 +4015,18 @@ ScriptEditor::ScriptEditor(WindowWrapper *p_wrapper) {
 	script_forward->set_disabled(true);
 	script_forward->set_tooltip_text(TTR("Go to next edited document."));
 
+	if (p_wrapper->is_window_available()) {
+		menu_hb->add_child(memnew(VSeparator));
+
+		make_floating = memnew(ScreenSelect);
+		make_floating->set_flat(true);
+		make_floating->set_tooltip_text(TTR("Make the script editor floating. test"));
```
Comment (YuriSizov) [inline editor/plugins/script_editor_plugin.cpp:4023]:
```suggestion
		make_floating->set_tooltip_text(TTR("Make the script editor floating."));
```
👀

**Review by YuriSizov — APPROVED** [review]
This is by popular demand as the button gets in the way more often than it is useful as is. So let's give it a go and then iterate more if needed.

File: editor/plugins/script_editor_plugin.cpp (hunk 1 above)
Comment (ajreckof) [reply editor/plugins/script_editor_plugin.cpp:4023]: Oups forgot to remove it 😅🤣. Just tested if I was at the right place. I'll remove it.

**Comment by YuriSizov** [comment]
Thanks!

**Pull request merged**